cmd/gophermart/main.go

```diff
@@ -1,3 +1,16 @@
 package main
 
-func main() {}
+import (
+	"log"
+
+	"GopherMart2/internal/router"
+)
+
+func main() {
+	rout := router.InitServer()
```

Comment:
```suggestion
	rout := &router.serverMart{}
```
какой смысл в этой функции, она все равно никакой логики не несет.